src/bip44.c

```diff
@@ -252,6 +255,42 @@ bool bip44_isMultidelegationStakingKeyPath(const bip44_path_t* pathSpec)
 	       && (bip44_getAddressValue(pathSpec) > 0);
 }
 
+bool bip44_isDRepKeyPath(const bip44_path_t* pathSpec)
+{
+#define CHECK(cond) if (!(cond)) return false
+	CHECK(pathSpec->length == BIP44_I_ADDRESS + 1);
```

Comment:
Nit: Why do functions above use `bip44_containsAddress(pathSpec)` and `!bip44_containsMoreThanAddress(pathSpec)` and not this?